easybuild/easyconfigs/s/Siesta/Siesta-4.0-intel-2017a.eb

```diff
@@ -0,0 +1,20 @@
+name = 'Siesta'
+version = '4.0'
+
+homepage = 'http://departments.icmab.es/leem/siesta'
+description = """SIESTA is both a method and its computer program implementation, to perform efficient electronic
+ structure calculations and ab initio molecular dynamics simulations of molecules and solids."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+vl = version.split('-')
+source_urls = ['https://launchpad.net/siesta/%s/%%(version)s/+download' % vl[0]]
+checksums = ['c4aa5f4cf2eabefaaacd3ef6350ec8fd']
+
+dependencies = [('netCDF-Fortran', '4.4.4')]
+
+parallel = 1
```

Comment:
@migueldiascosta This is no longer needed, right? It's handled by the easyblock?